cli.js

```diff
@@ -48,6 +48,19 @@ if (flags.uppercase) {
 	filename = filename.toUpperCase();
 }
 
+function readmeIsUpperCase() {
+	const results = globby.sync('readme.*', {nocase: true});
+	if (results.length > 0) {
+		const fileObj = path.parse(results[0]);
+		return fileObj.name.toUpperCase() === fileObj.name;
+	}
+	return false;
+}
+
+if (readmeIsUpperCase()) {
```

Comment:
This check could be combined with the flag support, which would allow the flag to short-circuit the synchronous globby call when provided:

```js
if (flags.uppercase || readmeIsUpperCase()) {
    filename = filename.toUpperCase();
}
```

Reply:
Nice catch, updated my PR to reflect this